client/layout/masterbar/woo.scss

```diff
@@ -2306,8 +2314,12 @@ $breakpoint-mobile: 660px;
 		@include link-woo;
 	}
 
-	.auth-form__social-buttons-tos a {
-		text-underline-offset: 2px;
+	.auth-form__social-buttons-tos {
+		color: var(--studio-gray-50) !important;
```

Comment:
Use `!important` due to:

![Screenshot 2025-01-23 at 12 23 16](https://github.com/user-attachments/assets/9bf8c8ac-46a5-4f08-86ea-bea9cf0a4123)